docs/core/responses.rst

```diff
@@ -156,3 +156,31 @@ The endpoint then needs to respond with the generated response:
   }
 
 Rasa will then use this response and sent it back to the user.
+
+
+.. _external-events:
+
+Proactively Reaching Out to the User with External Events
+---------------------------------------------------------
+
+You may want to proactively reach out to the user,
+for example to display the output of a long running background operation
+or notify the user of an external event.
+
+To do so, you can ``POST`` an intent to the
+`trigger_intent endpoint <../../api/http-api/#operation/triggerConversationIntent>`_.
+The intent, let's call it ``EXT_sensor``, will be treated as if the user had sent a message with this intent.
+You can even provide entities as parameters, e.g. ``{"temperature": "high"}``.
+For your bot to respond, we recommend you use the :ref:`mapping-policy` to connect the sent intent ``EXT_sensor``
+with the action you want your bot to execute, e.g. ``utter_warn_temperature``.
+You can also use a custom action here, of course.
+
+Use the ``output_channel`` query parameter to specify which output
+channel should be used to communicate the assistant's responses back to the user.
+Any messages that are dispatched in the custom action will be forwarded to the specified output channel.
```

Comment:
do you know if you can specify `latest` as the output channel here?

Reply:
I think @wochinge said the latest is automatically chosen if None is used.

Reply:
Nope, that's not true.
You can specify `latest` in case you want to use the latest input channel which the user used.

Reply:
default is always `CollectingOutputChannel`

Reply:
Ok, also lets specify how to send it to the latest channel here, bc it seems if you don't specify it comes back to you:
```
 If no output channel is specified, any messages that should be
        sent to the user will be included in the response of this endpoint.
```
i.e. if i have 2 channels, I want to just answer the user in the channel they used without having to query the tracker to get that info